xcell/mappers/mapper_Planck_base.py

```diff
@@ -61,6 +70,14 @@ def _get_mask(self):
         msk = hp.ud_grade(msk, nside_out=self.nside)
         return msk
 
+    def _get_ps_mask(self):
+        if self.ps_mask is None:
+            self.ps_mask = np.ones(hp.nside2npix(self.nside))
```

Comment:
I think `self.npix` exists in `MapperBase`

Reply:
done!

Reply:
I had to add it to base and removed it from SDSS_base